components/gc-table/gc-table-en.html

```diff
@@ -5,7 +5,7 @@
 		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" },
 		{ "title": "Provisional functionality", "link": "https://wet-boew.github.io/themes-dist/GCWeb/provisional-en.html" }
 	],
-	"dateModified": "2020-10-16",
+	"dateModified": "2021-06-09",
```

Comment:
Should the last review date on line 22 also be updated? Not sure what exactly it entails compared to date modified.

Technically any incoming PRs already need to be reviewed before merging. But if both dates are supposed to be one in the same, there wouldn't be any point in hardcoding last review's date if it could just refer to ``dateModified``.

Reply:
Yes, the date at line 22 should be updated too. You can use the jekyll variable
```
{{ page.dateModified }}
```
at line 22 to use the same date. You are free to apply it now or doing it in a other PR.